app/styles/main.scss

```diff
@@ -1 +1,122 @@
 // import "base";
+
+/* colors */
+
+$primary-color: #2196f3;
+$accent-color: #e84855;
+$footer-color: #403f4c;
```

Comment:
Better to create a _vars.scss in the same directory and importing it here to keep all the variables at one place.